Makefile

```diff
@@ -100,6 +100,8 @@ verify-commits:
 #
 # Example:
 #   make update
+#   update-generated-protobuf.sh must be before update-generated-completions.sh
```

Comment:
@smarterclayton the order is actually important. Also, the build is done before the needed files are generated. I needed to manually run the scripts before build worked. FYI. I made the change.